tests/Doctrine/Tests/Models/Forum/ForumCategory.php

```diff
@@ -9,7 +9,7 @@
 class ForumCategory
 {
     /**
-     * @Column(type="integer")
+     * @Column(type="guid", options={"fixed":true, "collation":"latin1_bin", "foo":"bar"})
```

Comment:
guid might not be a string so applying fixed or collation might not make sense in this case. maybe just use a string simply.